src/config/hooks/retrace.conf

```diff
@@ -0,0 +1,10 @@
+# Parameters are replaced using python's format.
+# Available parameters for cmdline: hook_name, taskid, task_results_dir
+# Example:
+# cmdline = {hook_name} {taskid} {task_results_dir}
+#
+# Before starting of the retracing itself
+# [pre_retrace_retrace]
```

Comment:
`pre_retrace_retrace` and `post_retrace_retrace` not consistent with the description at src/retrace/hooks/hooks.py:21,22

Reply:
"Typo", fixed.